script.js

```diff
@@ -33,8 +150,35 @@ function toggleMenu() {
 
 toggleBtn.addEventListener('click', toggleMenu);
 
-menuOptions.forEach((item) =>
-  item.addEventListener('click', () => {
-    if (!hamburgerImg.classList.contains('hamburger')) toggleMenu();
-  })
-);
+menuOptions.forEach((item) => item.addEventListener('click', toggleMenu));
+
+const portfolioButtons = document.querySelectorAll('.btn');
+portfolioButtons.forEach((button) => {
+  button.addEventListener('click', () => {
+    popup.classList.add('open-work-card');
+    console.log('pop up clicked!');
+  });
+});
+
+let formList = [];
+const addInfo = (ev) => {
+  ev.preventDefault();
+  let form = {
+    id: Date.now(),
+    name: document.getElementById('fname').value,
+    email: document.getElementById('email').value,
+    message: document.getElementById('subject').value,
+  };
+  formList.push(form);
+  document.forms[0].reset();
+
+  localStorage.setItem('myFormList', JSON.stringify(formList));
```

Comment:
- [ ] This data is not set to the local storage and upon reload, it throws an error. Attached is the error. Kindly consider fixing taking a second look at the code.
The setItem should take this format
```
localStorage.setItem('contactFormInput', JSON.stringify(formData));
```
![image](https://user-images.githubusercontent.com/86200356/202475511-3ccfa3df-7c2e-4a03-8ca6-215f05fe1aaa.png)

Reply:
Thank you for your feedback. Corrections has been made.